staubli_val3_driver/val3/ros_server/ros_server.dtx

```diff
@@ -455,5 +455,21 @@
         </Field>
       </Value>
     </Data>
+    <Data name="nVel" access="private" xsi:type="array" type="num" size="2" />
+    <Data name="nMsgState" access="private" xsi:type="collection" type="num">
+      <Value key="RecHeader" />
+      <Value key="DecHeader" value="1" />
+      <Value key="RecBody" value="2" />
+      <Value key="DecBody" value="3" />
+      <Value key="PushMotion" value="4" />
+      <Value key="Ack" value="5" />
+      <Value key="Reset" />
+    </Data>
+    <Data name="nGenericMsgType" access="private" xsi:type="collection" type="num">
```

Comment:
Suggestion: use the same name for this as in `simple_message`: `StandardMsgType` (from [here](https://github.com/ros-industrial/industrial_core/blob/d14029bdf0b403ef4412ef8a7b16bbb43a0a7627/simple_message/include/simple_message/simple_message.h#L58-L88)).